x-pack/test/fleet_api_integration/apis/epm/install_integration_in_multiple_spaces.ts

```diff
@@ -0,0 +1,113 @@
+import { skipIfNoDockerRegistry } from '../../helpers';
+import { setupFleetAndAgents } from '../agents/services';
+
+const testSpaceId = 'fleet_test_space';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const dockerServers = getService('dockerServers');
+  const server = dockerServers.get('registry');
+  const pkgName = 'system';
+  const pkgVersion = '1.27.0';
+
+  const installPackage = async (name: string, version: string) => {
+    await supertest
+      .post(`/api/fleet/epm/packages/${name}/${version}`)
+      .set('kbn-xsrf', 'xxxx')
+      .send({ force: true });
+  };
+
+  const uninstallPackage = async (pkg: string, version: string) => {
+    await supertest
+      .delete(`/api/fleet/epm/packages/${pkg}/${version}`)
+      .set('kbn-xsrf', 'xxxx')
+      .expect(200);
+  };
+
+  const installPackageInSpace = async (pkg: string, version: string, spaceId: string) => {
+    await supertest
+      .post(`/s/${spaceId}/api/fleet/epm/packages/${pkg}/${version}`)
+      .set('kbn-xsrf', 'xxxx')
+      .send({ force: true })
+      .expect(200);
+  };
+
+  const createSpace = async (spaceId: string) => {
+    await supertest.post(`/api/spaces/space`).set('kbn-xsrf', 'xxxx').send({
+      name: spaceId,
+      id: spaceId,
+      initials: 's',
+      color: '#D6BF57',
+      disabledFeatures: [],
+      imageUrl: '',
+    });
+  };
+
+  const deleteSpace = async (spaceId: string) => {
+    await supertest.delete(`/api/spaces/space/${spaceId}`).set('kbn-xsrf', 'xxxx').send();
+  };
+
+  const getTag = async (id: string, space?: string) =>
+    kibanaServer.savedObjects
+      .get({
+        type: 'tag',
+        id,
+        ...(space && { space }),
+      })
+      .catch(() => {});
+
+  describe('When installing system integration in multiple spaces', async () => {
+    skipIfNoDockerRegistry(providerContext);
+    setupFleetAndAgents(providerContext);
+
+    before(async () => {
+      if (!server.enabled) return;
+      await installPackage(pkgName, pkgVersion);
+
+      await createSpace(testSpaceId);
+      await uninstallPackage(pkgName, pkgVersion);
+      await installPackageInSpace(pkgName, pkgVersion, testSpaceId);
+    });
+
+    after(async () => {
+      await deleteSpace(testSpaceId);
+    });
+
+    it('should install kibana assets', async function () {
+      // These are installed from Fleet along with every package
+      const resIndexPatternLogs = await kibanaServer.savedObjects.get({
+        type: 'index-pattern',
+        id: 'logs-*',
+      });
+      expect(resIndexPatternLogs.id).equal('logs-*');
+      const resIndexPatternMetrics = await kibanaServer.savedObjects.get({
+        type: 'index-pattern',
+        id: 'metrics-*',
+      });
+      expect(resIndexPatternMetrics.id).equal('metrics-*');
```

Comment:
It would be great to check that the `installed_kibana_space_id` is set correctly on the SO. We have seen a support issue recently where this was the problem.